src/main/resources/gate.mixins.json

```diff
@@ -6,7 +6,7 @@
   "mixins": [
```

Comment:
I think we should move mixins from ``"mixins": []`` to ``"client": []`` to avoid them injecting if the mod is loaded server side

Reply:
(this is only a suggestion)

Reply:
Sure. I don’t know the difference so I don’t mind as long as it works